openedx/features/edly/tests/factories.py

```diff
@@ -0,0 +1,50 @@
+"""
+Model factories for unit testing views or models.
+"""
+
+from django.contrib.sites.models import Site
+from factory import SelfAttribute, Sequence, SubFactory
+from factory.django import DjangoModelFactory
+from organizations.tests.factories import OrganizationFactory
+
+from openedx.features.edly.models import EdlyOrganization, EdlySubOrganization
+
+
+class EdlyOrganizationFactory(DjangoModelFactory):
+    """
+    Factory class for Organization model
+    """
+
+    class Meta(object):
+        model = EdlyOrganization
+        django_get_or_create = ('name',)
+
+    name = Sequence('{}.testserver.fake'.format)
```

Comment:
nit: update it to something like `Edly Organization {}`

Reply:
Done